Description: Adds sed and awk binaries in the initramfs
 In Ubuntu, it seems that sed and awk are there by default, possibly in the
 built busybox. But in Debian, it's missing, and that is making growroot fail.
 This patch fixes it, until a better solution can be found.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Date: 2013-05-27

--- a/growroot/hooks/growroot	2013-05-26 22:47:14.000000000 +0800
+++ b/growroot/hooks/growroot	2013-05-27 00:02:58.000000000 +0800
@@ -12,5 +12,8 @@
 copy_exec /sbin/sfdisk /sbin
 copy_exec /usr/bin/growpart /sbin
 copy_exec /sbin/udevadm /sbin
+copy_exec /usr/bin/awk /bin
+copy_exec /bin/sed /bin
+copy_exec /bin/rm /bin
 
 # vi: ts=4 noexpandtab
